Cargo.toml

```diff
@@ -211,15 +211,17 @@ lto = "thin"
 # can wire up the 'abrt' signal handler so tempfiles will be removed in case of panics.
 panic = "unwind"
 incremental = false
-build-override = { opt-level = 0 }
 
 # This profile is currently used in building releases for GitHub.
 # It may be removed at any time and should not otherwise be relied on.
-[profile.release-opt]
+[profile.release-github]
 inherits = "release"
+overflow-checks = false
+panic = "unwind"
 lto = "fat"
 codegen-units = 1
 strip = "symbols"
+build-override = { opt-level = 3 }
```

Comment:
Assuming `build-override` is no longer to be set in the `release` profile from which this one inherits, isn't this superfluous? In the absence of `build-override`, I think build dependencies have the same `opt-level` as the profile, and since that is `release` here, we already have `opt-level = 3`.

You mentioned:

> As a side-note: The `inherits` directive is required even though I tried not to use it, so instead I repeated all values from the `release` profile to assure they are not actually inherited, but rather, controlled explicitly.

But only the values changed in `Cargo.toml` were repeated, except this, which is explicit here but implicit in the `release` profile. (`opt-level` for the profiles themselves is implicit in both.)

Reply:
With my recent gain of knowledge, I think this line should be removed. This was done in `main` now.